judgments/forms/search_forms.py

```diff
@@ -94,9 +94,10 @@ class AdvancedSearchForm(forms.Form):
     def clean(self):
         cleaned_data = super().clean()
         # Validate that from is before to now that we have access to both fields
-        # Cleaned data can never be `None`, so supress warning
-        to_date = cleaned_data.get("to_date")  # type: ignore
-        from_date = cleaned_data.get("from_date")  # type: ignore
+        if cleaned_data is None:  # make the type checker happier
+            raise RuntimeError("Cleaned data can never be None, this should never occur")
```

Comment:
Could we change this to a `return None` and keep the signature intact?
```suggestion
            return None
```

Reply:
Raising should keep the return value intact?